src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -3245,10 +3245,44 @@ LogicalResult ONNXGatherOp::inferShapes(
 //===----------------------------------------------------------------------===//
 // ConstantOfShape
 //===----------------------------------------------------------------------===//
+static LogicalResult verify(ONNXConstantOfShapeOp op){
```

Comment:
You will also need to change the signature of this method.

Reply:
What do I change the signature to?

Reply:
When I read your code, you have this
```
LogicalResult ONNXConstantOfShapeOp::verify() {
```

look at the other verify methods and do the same as them

Reply:
`LogicalResult classname::verify()` is the correct signature when using `let hasVerifier = 1;`.

@mmoldawsky updated the generation script to use the new pattern since direct usage of `verifier` will become obsolete soon. So all of the signatures need to change to the new pattern.

Reply:
Does that mean I have to go through each verifier and manually change the signature of each one?

Reply:
You'd have to also change things in the body of each verifier because in the new signature you don't have an op argument being passed in.